### ISSUE TITLE
Lucid Chart SVG import error

### ISSUE THREAD

**Issue by gtrott808** [ecceman/affinity#17]
Thank you for the SVG files! In Lucid SVG files come in as solid circles. I brought this issue up to Lucid, they had to modify the SVG in Inkscape to correct "path issues" they found with the files. It had to do with direction of paths. I've checked them with Affinity Designer and SVG debug websites. Couldn't  find any issues. Not sure what the issue is, they found. 

<img width="795" alt="Screen Shot 2021-06-20 at 10 14 13 AM" src="https://user-images.githubusercontent.com/12803985/122682938-08ae7680-d1b1-11eb-9d37-d8d6cb220553.png">

**Comment by mbhunters** [comment]
So what was the actual fix to get this in the "Fixed SVG" state?

**Comment by viptampa** [comment]
@gtrott808 looks like this is still an issue in 2023.   Any idea how users can fix this ourselves?

**Comment by jeffsmitty** [comment]
Yes, it still seems to be a problem in 2023 when adding some of these to a custom shape library.   Most of the "circle" SVG's come in as solid circles as show in the above comment, and some others seem to be parsed incorrectly by lucid.  In this image, you can see all the ones that came in a solid circles.  On the right, you can see the difference between adding from the shape library and dragging the SVG directly into lucid.  There's definitely either some inconsistency with the way some of these SVG's are constructed that causes lucid to not be able to import them properly.  
![image](https://github.com/ecceman/affinity/assets/7129042/b85fb004-b92c-40a6-abb4-dc6a52f6e925)
